Puppetfile

```diff
@@ -0,0 +1,4 @@
+#forge 'http://forge.puppetlabs.com'
+#
+#mod 'profiles', local: true
```

Comment:
I don't think you really need to use `local` in a Boltdir, since those should be in `site-modules` anyhow